data/week1/4-reading-input.md

```diff
@@ -162,8 +164,9 @@ Hi Lily
 </programming-exercise>
 
 ## Program Execution Waits for Input
-When the program's execution comes a statement that attempts to read input from the user (the command `reader.nextLine()`), the execution stops and waits. The execution continues only after the user has written some input and pressed enter.
-In the example below, the program prompts the user for three strings. First, the program prints `Write the first string: `, and then waits for user input. When the user writes some text, the program prints `Write the second string: `, and then waits for user input again. This continues for a third time, after which the program prints all three strings. Be aware, if the scanner is done receiving input, the scanner should be closed. If you do not close it, no error will pop up, but you will get deductions in style points anyway.
+When the program's execution reaches a statement that attempts to read input from the user (the command `reader.nextLine()`), the execution stops and waits. The execution continues only after the user has written some input and pressed enter.
+
+In the example below, the program prompts the user for three strings. First, the program prints `Write the first string: `, and then waits for user input. Once the user has entered a string and pressed enter, the program prints the next prompt and waits for another string. This process repeats for three strings in total. After the program has received all three strings, it prints them. Note that if the scanner is done receiving input, the scanner should be closed. If you don't close it, no error will be shown, but you will get deductions in style points anyway.
```

Comment:
'but you will get deductions in style points anyway.'

Dit kan weg, dat doen we namelijk niet.